web/src/components/organisms/CommentInput.jsx

```diff
@@ -0,0 +1,15 @@
+import { Label, Textarea } from 'flowbite-react';
+import React from 'react';
+
+const CommentInput = () => {
+    return (
+        <div className="col-start-1 col-end-auto">
+            <div className="mb-2 block">
+                <Label htmlFor="comment" value="" />
+            </div>
+            <Textarea id="comment" placeholder="Comment" required rows={4} />
```

Comment:
Remove the `id` and `required`
as id is not needed
for required, we will have yup validation for that, we dont want also make a component that is fixed to required, it is not really reusable if that is the case
```suggestion
            <Textarea placeholder="Comment" rows={4} />
```